ministryofjustice/analytics-platform-control-panel-frontend
# Use Redis for session storage

## What

* Use Redis for session storage to persist sessions across server restarts - meaning you don't have to repeatedly log in after changing a file in development
* Refactored app initialization to a modular style

## How to review

1. Run the app with `docker-compose up`
2. Login
3. Restart the server (by changing a file, for example)
4. Visit a page which requires login
5. See you don't have to login again
